pkgs/by-name/gl/glsl_analyzer/package.nix

```
@@ -19,6 +20,12 @@ stdenv.mkDerivation (finalAttrs: {
     zig_0_13.hook
   ];
 
+  buildInputs = lib.optionals stdenv.isDarwin [
```

```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
```

We’re deprecating this ambiguous form.

---

Ah, I do vaguely remember this. Fixed